plasmapy/analysis/nullpoint.py

```diff
@@ -978,16 +925,16 @@ def bound(epoint):
 
     tlApprox = trilinear_approx(vspace, cell)
 
-    # Check on the Surfaces
-    for p in BxByEndpoints:
-        if np.linalg.norm(tlApprox(p[0], p[1], p[2])) < _ATOL:
-            return True
-    for p in BxBzEndpoints:
-        if np.linalg.norm(tlApprox(p[0], p[1], p[2])) < _ATOL:
-            return True
-    for p in ByBzEndpoints:
-        if np.linalg.norm(tlApprox(p[0], p[1], p[2])) < _ATOL:
-            return True
+    # # Check on the Surfaces
+    # for p in BxByEndpoints:
+    #     if np.linalg.norm(tlApprox(p[0], p[1], p[2])) < _EQUALITY_ATOL:
+    #         return True
+    # for p in BxBzEndpoints:
+    #     if np.linalg.norm(tlApprox(p[0], p[1], p[2])) < _EQUALITY_ATOL:
+    #         return True
+    # for p in ByBzEndpoints:
+    #     if np.linalg.norm(tlApprox(p[0], p[1], p[2])) < _EQUALITY_ATOL:
+    #         return True
```

Comment:
It seems reasonable to me to leave this as a comment for the moment because we think we don't need it, but aren't sure.